### DISCUSSION TITLE
Inter DU handover successful with N310, issue in document and config file, and Grafana dashboard.

### DISCUSSION THREAD

**Discussion by asheeshtripathi** [srsran/srsRAN_Project#658] [Ideas & Suggestions]
I was able to successfully test the inter DU handover with USRP N310 and a COTS UE, both by moving the UE physically and chaning the Tx gains in runtime. However, the gNB config provided in the documentation is giving error when I run the gNB:
![image](https://github.com/srsran/srsRAN_Project/assets/48138145/6c344df7-4f3f-4442-82d4-87ac685b38a1)
![image](https://github.com/srsran/srsRAN_Project/assets/48138145/2c54949f-10ad-4429-999a-15f386497946)
 
 I changed the second report_configs: [ 1 ] to report_configs: [ 2 ] and was able to run the gNB with two cells and do the handover
![image](https://github.com/srsran/srsRAN_Project/assets/48138145/03d5c68e-32a8-44e0-bdf1-7b9fc3a929c3)
![image](https://github.com/srsran/srsRAN_Project/assets/48138145/dc7da88a-5528-4cc1-8dc4-c4261c675fd6)

Grafana Dashboard:  Dashboard most of the time shows cell count one(cell count becomes two intermittently, however two cells are configured for HO). As opposed to the UE count, which is most of the time one (with one UE attached) but becomes two during handovers.
![image](https://github.com/srsran/srsRAN_Project/assets/48138145/965f33e5-6f9b-427c-a442-b0222c115999)

**Comment by brendan-mcauliffe** [comment]
Hi @asheeshtripathi, 

We have updated the example configuration, it will be published tomorrow. The validator for the configuration file has recently been updated which resulted in the old configuration no longer being valid. Thank you for bringing this to our attention. 

As for the GUI, you momentarily see 2 UEs as there is a lag between the UE moving from one cell to the other making it momentarily appear as 2 UEs in Grafana. For the cells, we report the number of active cells, i.e. the number of cells with active UEs connected, so this is 1. For the same reason as seeing 2 UEs, you will sometimes see 2 cells.

**Reply by asheeshtripathi** [reply]
Thanks!
